src/main/java/ch/njol/skript/SkriptConfig.java

```diff
@@ -100,6 +101,8 @@ public void set(final Timespan t) {
 	
 	public final static Option<Boolean> usePlayerUUIDsInVariableNames = new Option<Boolean>("use player UUIDs in variable names", false); // TODO change to true later (as well as in the default config)
 	public final static Option<Boolean> enablePlayerVariableFix = new Option<Boolean>("player variable fix", true);
+
+	public final static Option<Boolean> caseInsensitiveVariables = new Option<Boolean>("case-insensitive variables", false);

```

Comment:
Should be true by default, would break Skript's default behaviour otherwise (setting `{_SomEthing}` will actually set `{_SomEthing}` and not `{_something}` as intended.